WordPress/build.gradle

```diff
@@ -144,7 +148,10 @@ android {
             buildConfigField "String", "PUSH_NOTIFICATIONS_APP_KEY", '"com.jetpack.android"'
```

Comment:
I see there there's a manually-overridden `buildConfigField` here – would it be possible to read the `jp.sentry.dsn` value here (vs specifying it a string)?

This might be a bad plan – @oguzkocer could speak to it, just a thought I had while reading through this

Reply:
I think this would have been a fine temporary solution mainly because I think we'll have to re-work these `buildConfigField` declarations in Android Gradle Plugin 4.2 upgrade. However, sentry dsn is different from the other overrides because it's a secret and we'd have to load up the gradle properties file even if we only use the sentry dsn value. If we are going to do that, we might as well go for more generic/stable solution.